src/pages/IstioConfigDetails/IstioRuleInfo.tsx

```diff
@@ -162,10 +156,7 @@ class IstioRuleInfo extends React.Component<IstioRuleInfoProps> {
             formatters: [this.headerFormat]
           },
           cell: {
-            formatters: [this.cellFormat],
-            props: {
-              align: 'text-center'
-            }
+            formatters: [this.cellFormat]
```

Comment:
where does it take the alignment style from? I see you deleted the text-center, so the left alignment is taken from somewhere else?

Reply:
It is the default alignment style from the component.
Is it looking different locally to you ?